### DISCUSSION TITLE
Using fontawesome alongside with heroicons

### DISCUSSION THREAD

**Discussion by monzer15** [filamentphp/filament#13073] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.80

### How can we help you?

Hi,
I would like to use font awesome without replacing the default icons.

i followed the instructions from: [https://v2.filamentphp.com/tricks/use-font-awesome-or-any-other-icon-set](url) and installed owenvoke/blade-fontawesome.

but I am getting this error : Svg by name "fa-solid fa-list-check" from set "default" not found.

**Comment by leandrocfe** [comment, marked as answer]
You should use the name of the icon `'fas-list-check'`

**Reply by monzer15** [reply]
thanks!
